### ISSUE TITLE
Migrate results in 'Meta' object has no attribute 'private_fields' if using django-constance

### ISSUE THREAD

**Issue by bb4L** [tolomea/django-data-browser#22]
Hi I tried to use this package in a existing project

when running `python manage.py migrate` it fails with `AttributeError: 'Meta' object has no attribute 'private_fields'`
in 
```
File "/usr/local/lib/python3.6/site-packages/django/forms/models.py", line 143, in fields_for_model
    sortable_private_fields = [f for f in opts.private_fields if isinstance(f, ModelField)]
```

This happens while applying `data_browser.0009_migrate_saved_views`

This is a issue if you use [django-constance](https://github.com/jazzband/django-constance)

i was able to resolve it with:

```
from constance.admin import ConstanceAdmin, Config

class ConfigAdmin(ConstanceAdmin):
    ddb_ignore = True

admin.site.unregister([Config])
admin.site.register([Config], ConfigAdmin)
```

**Comment by tolomea** [comment]
Thanks for the report.
I will take a look into this. Would you be able to paste the complete stack trace?
Also what versions of Django and Constance are you using?

**Comment by bb4L** [comment]
Django: 2.2.17
django-constance: 2.8.0

```
Running migrations:
  Applying data_browser.0001_initial... OK
  Applying data_browser.0002_auto_20200331_1842... OK
  Applying data_browser.0003_remove_view_app... OK
  Applying data_browser.0004_auto_20200501_0903... OK
  Applying data_browser.0005_auto_20200516_1726... OK
  Applying data_browser.0006_auto_20200531_1450... OK
  Applying data_browser.0007_view_public_slug... OK
  Applying data_browser.0008_view_limit... OK
  Applying data_browser.0009_migrate_saved_views...Traceback (most recent call last):
  File "manage.py", line 21, in <module>
    main()
  File "manage.py", line 17, in main
    execute_from_command_line(sys.argv)
  File "/home/vscode/.local/lib/python3.6/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/home/vscode/.local/lib/python3.6/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/vscode/.local/lib/python3.6/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/vscode/.local/lib/python3.6/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/home/vscode/.local/lib/python3.6/site-packages/django/core/management/base.py", line 83, in wrapped
    res = handle_func(*args, **kwargs)
  File "/home/vscode/.local/lib/python3.6/site-packages/django/core/management/commands/migrate.py", line 234, in handle
    fake_initial=fake_initial,
  File "/home/vscode/.local/lib/python3.6/site-packages/django/db/migrations/executor.py", line 117, in migrate
    state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
  File "/home/vscode/.local/lib/python3.6/site-packages/django/db/migrations/executor.py", line 147, in _migrate_all_forwards
    state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
  File "/home/vscode/.local/lib/python3.6/site-packages/django/db/migrations/executor.py", line 245, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/home/vscode/.local/lib/python3.6/site-packages/django/db/migrations/migration.py", line 124, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/home/vscode/.local/lib/python3.6/site-packages/django/db/migrations/operations/special.py", line 190, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "/home/vscode/.local/lib/python3.6/site-packages/data_browser/migrations/0009_migrate_saved_views.py", line 10, in forwards
    forwards_0009(View)
  File "/home/vscode/.local/lib/python3.6/site-packages/data_browser/migration_helpers.py", line 56, in forwards_0009
    models = get_models(request)
  File "/home/vscode/.local/lib/python3.6/site-packages/data_browser/orm_admin.py", line 312, in get_models
    model_admins, admin_fields = _get_all_admin_fields(request)
  File "/home/vscode/.local/lib/python3.6/site-packages/data_browser/orm_admin.py", line 134, in _get_all_admin_fields
    get_common(model_admin, model)
  File "/home/vscode/.local/lib/python3.6/site-packages/data_browser/orm_admin.py", line 124, in get_common
    all_admin_fields[model].update(from_fieldsets(admin, False))
  File "/home/vscode/.local/lib/python3.6/site-packages/data_browser/orm_admin.py", line 95, in from_fieldsets
    fields = admin.get_fieldsets(request, obj)
  File "/home/vscode/.local/lib/python3.6/site-packages/django/contrib/admin/options.py", line 330, in get_fieldsets
    return [(None, {'fields': self.get_fields(request, obj)})]
  File "/home/vscode/.local/lib/python3.6/site-packages/django/contrib/admin/options.py", line 321, in get_fields
    form = self._get_form_for_get_fields(request, obj)
  File "/home/vscode/.local/lib/python3.6/site-packages/django/contrib/admin/options.py", line 659, in _get_form_for_get_fields
    return self.get_form(request, obj, fields=None)
  File "/home/vscode/.local/lib/python3.6/site-packages/django/contrib/admin/options.py", line 705, in get_form
    return modelform_factory(self.model, **defaults)
  File "/home/vscode/.local/lib/python3.6/site-packages/django/forms/models.py", line 551, in modelform_factory
    return type(form)(class_name, (form,), form_class_attrs)
  File "/home/vscode/.local/lib/python3.6/site-packages/django/forms/models.py", line 256, in __new__
    apply_limit_choices_to=False,
  File "/home/vscode/.local/lib/python3.6/site-packages/django/forms/models.py", line 142, in fields_for_model
    sortable_private_fields = [f for f in opts.private_fields if isinstance(f, ModelField)]
AttributeError: 'Meta' object has no attribute 'private_fields'

```

**Comment by tolomea** [comment]
I've found a tidy way to detect this particular form of shenanigans. I have released that as 3.0.4
